includes/Context.php

```diff
@@ -510,4 +510,26 @@ public function get_locale( $context = 'site', $format = 'default' ) {
 
 		return $wp_locale;
 	}
+
+	/**
+	 * Check whether the site has auto updates enabled for Site Kit.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool `true` if auto updates are enabled, otherwise `false`.
+	 */
+	public function is_autoupdate_enabled() {
+		if ( $this->is_network_active() ) {
+			$enabled_auto_updates = (array) get_site_option( 'auto_update_plugins', array() );
+		} else {
+			$enabled_auto_updates = (array) get_option( 'auto_update_plugins', array() );
+		}
```

Comment:
Is this needed? Looking at `wp_ajax_toggle_auto_updates`, it uses `*_site_option` functions.

Reply:
No, its not. Will change it to only use `get_site_option`. Wasn't aware, that `get_site_option` on single site install is the same as `get_option`.